src/main/resources/emails/emailReport.ftl

```diff
@@ -469,8 +469,8 @@ ${sender.name} sent you a report from ANET:
     </div>
   </#if>
 
-  <#if report.loadApprovalStatus()??>
-    <#assign approvalStatus = report.loadApprovalStatus()>
+  <#if report.loadApprovalStatus(context)??>
```

Comment:
I think you should use report.loadApprovalStatus(context).get(), as loadApprovalStatus returns a CompletableFuture.